aiohttp/payload.py

```diff
@@ -50,19 +61,26 @@ class PayloadRegistry:
     """
 
     def __init__(self):
-        self._registry = []
+        self._first = []
+        self._normal = []
+        self._last = []
 
-    def get(self, data, *args, **kwargs):
+    def get(self, data, _CHAIN=chain, *args, **kwargs):
```

Comment:
Shouln't `_CHAIN`  be keyword-only argument? Otherwise it will be overrided by a second argument which should get into `args`.

Reply:
good catch